### ISSUE TITLE
Input error when read in .mod file with .datainfo file

### ISSUE THREAD

**Issue by liuxl17** [pharmpy/pharmpy#952]
Sometimes the .mod file cannot read in $DATA properly when having .datainfo file with the same name as dataset under the same directory. The errors are specified as below:

1. Column missing
![image](https://user-images.githubusercontent.com/53524647/177876263-ed4c2d28-e02c-4106-b173-24617106a3b2.png)

2. Fail to read in the .mod file
![image](https://user-images.githubusercontent.com/53524647/177877033-fe9ab1e6-dce2-41ea-9b85-a59fbff1a372.png)

**Comment by pharmpy-dev-123** [comment]
Hi @liuxl17. The first problem that shows some columns missing might be because the `.datainfo` file does not list all the columns. Can you check that the `.datainfo` file lists all columns? The second problem might be a bug.

**Comment by liuxl17** [comment]
Yes, the ```.datainfo``` file lists all columns, and it is converted from ```.csv``` file using the webtool provided in the User guide-Datasets in Pharmpy-DataInfo (Notes) (https://pharmpy.github.io/amdui/datainfo/)

**Comment by ZheHuang96** [comment]
These errors are interesting, our current strategy is to let $input to override the datainfo. Xuanlin, could you send me the model and datainfo file via slack or whatever you prefer, I would like to reproduce and see what happened.
